dumb-jump.el

```diff
@@ -3037,7 +3037,10 @@ Using ag to search only the files found via git-grep literal symbol search."
 			   0)))
 		       (if do-var-jump
 			   (list var-to-jump)
-			 match-cur-file-front)))))))
+             match-cur-file-front))))))
+
+    (cl-defmethod xref-backend-apropos ((_backend (eql dumb-jump)) symbol)
```

Comment:
The generic method names the second parameter `pattern` (some backends choose `regexp`). I think it would be use a term along these lines to make clear that the argument isn't generated by `xref-backend-identifier-at-point`, and therefore has no context.

Reply:
Done